index.html

```diff
@@ -368,6 +368,25 @@ <h2>
                 <p>Return <var>p</var>.</p>
               </li>
             </ol>
+            <p>
+              The <a>user agent</a> MUST NOT capture content from the background of a
+              partially transparent captured <a>display surface</a>.
+            </p>
+            <p>
+              In the case the user prompt displays previews of the various surfaces available for selection,
```

Comment:
Can we simplify as MUST NOT capture any previews?

Reply:
I think that's too broad. If the app is already capturing the monitor and then calls getDisplayMedia again, would you expect UAs to remove previews from the monitor capture? That seems arduous. I think the intent here was that UAs be careful about starting capture after the related prompt has closed.

Reply:
Note "for the newly created track" below. That means that if the UA is already capturing the entire screen, it's irrelevant - that's a pre-existing track.

Reply:
You could in theory have prompts up for multiple getDisplayMedia calls, so I feel it's good to spell out that this is related to the prompt at hand.

Reply:
I agree, but I don't think the current language conveys that. It currently says that the user agent MUST NOT capture "the previews that the user did not intend to share explicitly." This conjures in my mind the question of "which previews *did* the user intend to share, and what's the difference between sharing and explicitly sharing?" It seems that this was intended as a clarification of the rationale, but it's not as clear to me as it may be to the original author. So I think it would be clearer to say something along the lines of "MUST NOT capture the previews that were part of the prompt" or something else.

Reply:
Agree. This is existing language that's only been moved, so I wanted to be careful to keep this editorial. But clarity is desirable, so applying the requirement to all previews (including the one the user chose) seems like a reasonably practical interpretation that I doubt any implementation would object to. Let me fix it.